src/libunwind/lib.rs

```diff
@@ -22,7 +22,11 @@ cfg_if::cfg_if! {
     }
 }
 
-#[cfg(target_env = "musl")]
+#[cfg(all(not(target_arch = "hexagon"), target_env = "musl"))]
 #[link(name = "unwind", kind = "static", cfg(target_feature = "crt-static"))]
 #[link(name = "gcc_s", cfg(not(target_feature = "crt-static")))]
 extern {}
+
+#[cfg(all(target_arch = "hexagon", target_env = "musl"))]
+#[link(name = "unwind", kind = "static", cfg(target_feature = "crt-static"))]
```

Comment:
This seems slightly suspicious in the sense that the above should work even for the hexagon target I think? Is it possible to remove this change, and if not what does the error look like?

Reply:
Sorry, I think this is a stale change that was merged from an older baseline where it made some sense.

I'll remove it.